docs/dev/connectors/streamfile_sink.md

```diff
@@ -204,6 +205,65 @@ input.addSink(sink)
 </div>
 </div>
 
+#### Avro format
+
+Flink also provides built-in support for writing data into Avro files. A list of convenience methods to create
+Avro writer factories and their associated documentation can be found in the 
+[AvroWriters]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/formats/avro/AvroWriters.html) class.
+
+For creating customized Avro writers like enabling compression, users need to create the `AvroWriterFactory`
+with a custom implementation of the [AvroBuilder]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/formats/avro/AvroBuilder.html) interface.
```

Comment:
Would it make sense to add an example on creating customized Avro writers?

Reply:
I have added the example for the customized Avro writers.